curriculum/challenges/english/01-responsive-web-design/basic-css/change-a-variable-for-a-specific-area.md

```diff
@@ -23,7 +23,7 @@ The `penguin` class should reassign the `--penguin-belly` variable to `white`.
 
 ```js
 assert(
-  code.match(/.penguin\s*?{[\s\S]*\n\s*--penguin-belly\s*?:\s*?white\s*?;[\s\S]*}/gi)
+  code.match(/\.penguin\s*?{[\s\S]*(?<!background-color: )--penguin-belly\s*?:\s?white\s*?;[\s\S]*}/gi)
```

Comment:
Not sure if negative lookbehinds are supported across all the browsers we support currently.  I am also not sure if Babel will transform it into something that can work.  We have no other challenges in the curriculum that use a negative lookbehind in a challenge test.

Reply:
Maybe another test won't hurt:

your `.penguin` class should not contain `background-color`

Test:
`code.match(/^((?!background-color:).)*$/gs)`

Reply:
@RandellDawson

Reply:
We currently only support Firefox and Chrome (both of which can [handle it](https://caniuse.com/?search=lookbehind)).  As far as Babel goes, we don't do anything to the tests themselves.

Reply:
This is a solution that also works, but I've also made another suggestion.